packages/puppeteer-core/src/cdp/Browser.ts

```diff
@@ -211,6 +219,11 @@ export class CdpBrowser extends BrowserBase {
         proxyBypassList: proxyBypassList && proxyBypassList.join(','),
       },
     );
+    await this.#connection.send('Browser.setDownloadBehavior', {
```

Comment:
let's move this into a method on the CdpBrowserContext and let's call when we create a new context here and the second call let's place it in CdpBrowser._attach when the default context is created.